src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 3; i++) {
+            Ball randomBall = lottery.getRandomBall();
+            String ballString = randomBall.toString();
+            System.out.println(ballString);
```

Comment:
System.out.println automatically calls .toString() method of object passed, no need to explicitly call it
```suggestion
            System.out.println(lottery.getRandomBall());
```